src/delay.js

```diff
@@ -67,17 +67,15 @@ define(function (require) {
      *  <a href="http://www.w3.org/TR/webaudio/#DelayNode">
      *  Web Audio Delay Nodes</a>, one for each stereo channel.
      *
-     *  @property leftDelay
-     *  @type {Object}  Web Audio Delay Node
+     *  @property leftDelay {Object}  Web Audio Delay Node
```

Comment:
hmm, it seems I should get rid of the type descriptions because they wind up as part of the property name 

![screenshot 2017-07-12 00 05 42](https://user-images.githubusercontent.com/504124/28101343-eed8fb5c-6695-11e7-9bc1-357035b6581b.png)